src/components/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect, useContext } from "react";
+import { useNavigate } from "react-router-dom";
+import "./SideBar.css";
+import { Link } from "react-router-dom";
+import { AuthContext } from "../../contexts/spotifyContext";
+
+function SideBar() {
+  //const [accessToken, setAccessToken] = useState(null);
+  const [userData, setUserData] = useState({
+    display_name: "",
+    images: [],
+  });
+  const accessToken = localStorage.getItem("spotify_access_token");
+  useEffect(() => {
+    // const accessToken = localStorage.getItem("spotify_access_token");
+    console.log(accessToken);
+    if (accessToken) {
+      fetch("https://api.spotify.com/v1/me", {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+        },
+      })
```

Comment:
* Please, define the api functions inside `SpotifyApi.js`, and then import them into the components to separtate the logics